BUILD/mdbci/build.sh

```diff
@@ -66,9 +66,7 @@ $(<${script_dir}/templates/build.json.template)
 
 	# destroying existing box
 	if [ -d "$MDBCI_VM_PATH/${name}" ]; then
-		cd $MDBCI_VM_PATH/${name}
-		vagrant destroy -f
-		cd ${dir}
+                ${mdbci_dir}/mdbci destroy $name
```

Comment:
According to the test on 68th string, you check for configuration in the `$MDBCI_VM_PATH` directory, but here you relative path to the configuration. I think that absolute path is necessary here too.

Reply:
$MDBCI_VM_PATH is in use by default.  $name is a name of configuration, not a path. If MDBCI uses some other way to interpret this parameter it have to be fixed